Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -859,6 +959,578 @@ def add_custom_ip_feeds(client: PrismaCloudComputeClient, args: dict) -> Command
+    """
+    Add custom md5 hashes of malware to the prisma cloud compute.
+    Implement the command 'prisma-cloud-compute-custom-feeds-malware-add'
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): prisma-cloud-compute-custom-feeds-malware-add command arguments.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    # the api overrides the md5 malware hashes, therefore it is necessary to add those who exist to the 'PUT' request.
+    feeds = client.get_custom_md5_malware().get("feed", [])
+    name, md5s = args.get("name"), argToList(arg=args.get("md5", []))
+
+    existing_md5s = set([feed.get("md5") for feed in feeds])
+    for md5 in md5s:
+        if md5 not in existing_md5s:  # verify that there are no duplicates because the api doesn't handle it
+            feeds.append({"name": name, "md5": md5})
+
+    client.add_custom_md5_malware(feeds=feeds)
+
+    return CommandResults(readable_output="Successfully updated the custom md5 malware feeds")
+
+
+def get_cves(client: PrismaCloudComputeClient, args: dict) -> List[CommandResults]:
+    """
+    Get cves information, implement the command 'cve'.
+
+    Args:
+        client (PrismaCloudComputeClient): prisma-cloud-compute client.
+        args (dict): cve command arguments.
+
+    Returns:
+        CommandResults: command-results object.
+    """
+    cve_ids = argToList(arg=args.get("cve_id", []))
+    all_cves_information, results, unique_cve_ids = [], [], set()
+
+    for _id in cve_ids:
+        if cves_info := client.get_cve_info(cve_id=_id):
+            all_cves_information.extend(cves_info)
+
+    if filtered_cves_information := filter_api_response(api_response=all_cves_information, limit=MAX_API_LIMIT):
+        for cve_info in filtered_cves_information:
+            cve_id, cvss = cve_info.get("cve"), cve_info.get("cvss")
+            modified, description = epochs_to_timestamp(epochs=cve_info.get("modified")), cve_info.get("description")
+
+            if cve_id not in unique_cve_ids:
+                unique_cve_ids.add(cve_id)
+
+                cve_data = {
+                    "ID": cve_id, "CVSS": cvss, "Modified": modified, "Description": description
+                }
+
+                results.append(
+                    CommandResults(
+                        outputs_prefix="CVE",
+                        outputs_key_field=["ID"],
+                        outputs=cve_data,
+                        indicator=Common.CVE(
+                            id=cve_id, cvss=cvss, published="", modified=modified, description=description
+                        ),
+                        raw_response=filtered_cves_information,
+                        readable_output=tableToMarkdown(name=cve_id, t=cve_data)
+                    )
+                )
+
+        return results
+
+    return [CommandResults(readable_output="No results found")]
```

Comment:
it is possible but not needed, there is a return statement inside the `if` and in case the code block does not enter the `if` statement 

two examples which are basically the same:

` if a:
        return b
    return c
    
    if a:
        return b
    else:
        return c`

Reply:
> Looks good! 👍
> 
> 1. As we discussed, let's add to the human-readable in `get-impacted-resources` also the CVE so we can see which cve impacted what.
> 2. Please add all the arguments that missing (where using `assign_params`) for better readability.
> 3. Add to the description of the argument (like id) where can we get it from.
> 4. Make sure that the `cluster` argument, indeed works for all the commands.

Thank you for the quick code review :-)

1) done
2) done
3) done that for every parameter, there are some who cannot be retrieved from previous commands, but can be retrieved by running the same command without any filter parameters
4) basically right now in our test environment there isn't a k8s cluster, but i verified that on a different environment and it works as expected in all of the commands that use that. In addition its main purpose is simply to filter responses most of the time